src/test/crimson/CMakeLists.txt

```diff
@@ -22,12 +22,6 @@ target_link_libraries(unittest_seastar_messenger crimson)
 add_executable(test_seastar_messenger_peer test_messenger_peer.cc)
 target_link_libraries(test_seastar_messenger_peer ceph-common global ${ALLOC_LIBS})
 
-add_executable(perf_crimson_msgr perf_crimson_msgr.cc)
```

Comment:
i am fine with this change. but why?

Reply:
I need to measure crimson msgr performance with `WITH_TESTS=OFF`, but it will disable all targets defined in `src/test/`. So move it to `src/tools/crimson/`

Reply:
I am curious why you’d want disable WITH_TESTS? What’s the merit of doing so?

Reply:
Like async msgr, `WITH_TESTS` enables `UNIT_TESTS_BUILT`, which will compile intercept logic with unnecessary overhead, see:
* https://github.com/ceph/ceph/blob/master/src/crimson/net/ProtocolV2.cc#L130-L155
* https://github.com/ceph/ceph/blob/master/src/msg/async/ProtocolV2.cc#L64-L77